### PR TITLE
Improve on Scramble integration

### PR DESCRIPTION
**Pull request by VJalili**
This PR improves on the integration of the scramble with the pipeline by adding missing references to the Scramble calls.

### CODE REVIEW

**Review by epiercehoffman — CHANGES_REQUESTED** [review]
Thanks for working on this, Vahid. It looks like the changes to EvidenceQC are complete, so they just need to be tested. For right now I think we want to focus on the canonical pipeline, and Module10 is harder to test, so I would leave Module10 for a later time (maybe we should open a ticket for further Scramble integration tasks? I also had some listed in #331 that we will want to come back to at a later date).

Some additional changes that are needed to enable running GATK-SV with Scramble in Terra in cohort mode:
* Add scramble docker to the Terra cohort mode workspace.tsv template (then check that properly formatted TSV is generated by build_default_inputs.sh, since this is not covered by a validation script)
* Add scramble inputs to Terra cohort mode JSON templates for: EvidenceQC, GatherBatchEvidence, ClusterBatch, GenerateBatchMetrics, FilterBatchSites, PlotSVCountsPerSample, FilterBatchSamples

Please let me know when you've completed testing (womtool validation, Terra JSON validation, & run any WDLs that were modified).

**Comment by VJalili** [comment]
Thanks @epiercehoffman for reviewing this. I updated the templates as you suggested and successfully run `EvidenceQC` on the _methods_ Cromwell using 1kgp sample data (the UUID of the run is `8dab36d2-9cf7-4c48-b626-8fccd8996e2d`).

File: dockerfiles/rdpesr/integrate_annotations.R [hunk 1]
```diff
@@ -102,4 +102,4 @@ vs_melt=read.table(opt$raw_melt, comment.char="", header=T, sep='\t')
 colnames(vs_melt)[c(4,8,9)]=c('SVID','vs_raw_melt_ovr1a','vs_raw_melt_ovr1b')
 dat=merge(dat, vs_melt[,c(4,8,9)], by='SVID')
 
-write.table(dat[,c(2:4,1,5:ncol(dat))],opt$output, quote =F, sep='\t', col.names=T, row.names=F)
```
Comment (mwalker174) [inline dockerfiles/rdpesr/integrate_annotations.R:105]: I would revert this to keep the history clean

File: inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl [hunk 2]
```diff
@@ -27,5 +27,6 @@
   "ClusterBatch.wham_vcf_tar": "${this.std_wham_vcf_tar}",
   "ClusterBatch.manta_vcf_tar": "${this.std_manta_vcf_tar}",
   "ClusterBatch.melt_vcf_tar": "${this.std_melt_vcf_tar}",
+  "ClusterBatch.scramble_vcf_tar": "${this.std_scramble_vcf_tar}",
```
Comment (mwalker174) [inline inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl:30]: We don't want to actually add scramble to any of the templates until we've evaluated it more thoroughly (also when we do migrate to scramble we'll want to disable melt and delete the melt inputs across workflows). Can you revert the changes to all `.tmpl` files? This includes the update to the terra workspace variable definitions.

**Review by mwalker174 — CHANGES_REQUESTED** [review]
Thanks for working on this @VJalili. We actually want to hold off on adding scramble to tests until it's evaluated (see comment). Also make sure you run a full validation with this command:
```
bash scripts/test/validate.sh -d . -j  /path/to/womtool-65.jar -t
```
The `-t` will run Terra inputs validation, which is currently failing (although it may pass when you revert the templates):
```
ERROR: Unexpected input: ./inputs/build/ref_panel_1kg/terra/workflow_configurations/EvidenceQC.json contains unexpected input EvidenceQC.scramble_vcfs for EvidenceQC.wdl
```

File: inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl (hunk 2 above)
Comment (VJalili) [reply inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl:30]: I changed the `.tmpl` following @epiercehoffman suggestions and I think the motivation was to enable running scramble benchmarks on Terra. Happy to revert these changes if we want to hold off on them for a while.

Comment (epiercehoffman) [reply inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl:30]: @mwalker174 I want to add scramble to the Terra templates except for GatherSampleEvidence, so that it can be toggled on/off just by adding/removing the scramble docker in GatherSampleEvidence, just like MELT. But I agree we don't want to add it to the test JSONs just yet, so please revert the changes to `inputs/templates/test/EvidenceQC/EvidenceQC.json.tmpl`.

**Comment by VJalili** [comment]
@mwalker174 I reverted the changes discussed above and ran the validation script as you suggested. The script ran successfully with the following output.

```
Success! All Terra input JSONs passed validation.
Pass: 31. Fail: 0.
Total: 31 out of 31 passed.
```

**Comment by epiercehoffman** [comment]
@VJalili I think there was some confusion - can you remove the scramble changes from `inputs/templates/test/EvidenceQC/EvidenceQC.json.tmpl` but keep them for the Terra JSON template?

**Comment by VJalili** [comment]
@epiercehoffman I modified the wrong file, please see the last commit where I fixed it.

**Review by epiercehoffman — APPROVED** [review]
Thanks, looks good to me!

File: inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl (hunk 2 above)
Comment (mwalker174) [reply inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl:30]: @VJalili Just discussed this with emma. Fine to keep these Terra workflow configs in place since it doesn't actually enable scramble (ie providing the scramble docker in GatherSampleEvidence).

**Review by mwalker174 — APPROVED** [review]

**Comment by VJalili** [comment]
Thank you both for reviewing this.
